data/Ubuntu-16.04.yaml

```diff
@@ -0,0 +1,2 @@
+---
+nscd::service_restart: /usr/sbin/service nscd reload
```

Comment:
is this valid? I never saw a 16.04 without systemd and in all other modules we assume 16.04 and newer ubuntus are running with systemd

Reply:
`/bin/systemctl` just to be annoying.